scripts/parallel_deblur

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015, The Deblur Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import click
+from os.path import join, exists, split
+from os import makedirs, mkdir
+from glob import glob
+from shutil import rmtree
+
+from skbio.util import remove_files
+
+from deblur.parallel_deblur import parallel_deblur
+from deblur.workflow import (merge_otu_tables,
+                             split_sequence_file_on_sample_ids_to_files)
+
+
+@click.group()
+def parallel_deblur_cmds():
+    pass
+
+
+# LAUNCH FULL DEBLUR PIPELINE COMMAND
```

Comment:
nonblocking, but much of the help text and option arguments seem repeated. it may be useful in the future to explore a decomposition